tests/account_change_password_test.go

```diff
@@ -120,7 +120,7 @@ func changePassword(t *testing.T, passwordClient *client.CSAPI, oldPassword stri
 	})
 }
 
-func createSession(t *testing.T, deployment *docker.Deployment, userID, password string) *client.CSAPI {
+func CreateSession(t *testing.T, deployment *docker.Deployment, userID, password string) *client.CSAPI {
```

Comment:
These don't need to be public. Lower case them please.

Reply:
Do we have to write them again in the pushers test file?

Reply:
No, it's all in the same package.

Reply:
Fixed